### PR TITLE
WIP: Utils & CLI

### PR DESCRIPTION
**Pull request by petrbel**
**Please squash**

* [X] standard RethinkDB functions (`utils` module)
* [X] basic CLI
* [x] fix current tests
* [ ] allow variable filtering
* [ ] test `utils`
* [ ]  advanced functions

### CODE REVIEW

File: cxflow_rethinkdb/rethinkdb_hook.py [hunk 1]
```diff
@@ -58,7 +60,7 @@ class RethinkDBHook(AbstractHook):
```
Comment (FloopCZ) [inline cxflow_rethinkdb/rethinkdb_hook.py]: Please update the `example usage in config` to correspond to `cxflow-1.0`.

File: cxflow_rethinkdb/cli.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    select_by_id_parser = subparsers.add_parser('select-by-id')
+    select_by_id_parser.set_defaults(subcommand='select-by-id')
+    select_by_id_parser.add_argument('db_name', help='name of the db from which documents will be selected')
+    select_by_id_parser.add_argument('table_name', help='name of the table from which documents will be selected')
+    select_by_id_parser.add_argument('id', help='document ID')
+
+    # add common arguments
+    for parser in [main_parser, create_db_parser, create_table_parser, create_user_parser, grant_permission_parser,
+                   insert_parser, select_all_parser, select_by_id_parser]:
+        parser.add_argument('-c', '--credentials', help='path to the credentials file')
```
Comment (FloopCZ) [inline cxflow_rethinkdb/cli.py]: I am just thinking - cannot we make the credentials file to be an unnamed argument? It is always needed anyway.

File: README.md [hunk 3]
```diff
@@ -24,3 +24,126 @@ pip install cxflow-rethinkdb
 When `cxflow-rethinkdb` installed, the following classes are available:
 
 - `cxflow_rethinkdb.RethinkDBHook`
+
```
Comment (FloopCZ) [inline README.md]: I would also like to see an example usage in cxflow's config.yaml as described in the RethinkDBHook's docstring.

**Comment by petrbel** [comment]
Closing this for now as lots have changed. I'll keep on developing someday, hopefully.

**Pull request closed without merging**